logrotate/test/RotateFileTest.cpp

```diff
@@ -77,8 +77,27 @@ TEST_F(RotateFileTest, setMaxLogSize) {
 
    exists = Exists(content, first_message_in_new_log);
    EXPECT_TRUE(exists) << "\n\tcontent:" << content << "-\n\tentry: " << gone;
+}
+
+TEST_F(RotateFileTest, setMaxLogSizeAndRotate) {
```

Comment:
Can you add a unit test for empty file name since that logic is slightly changed?